odoo/addons/base/views/ir_sequence_views.xml

```diff
@@ -10,6 +10,7 @@
                       <group>
                         <field name="name"/>
                         <field name="implementation"/>
+                        <field name="sequence_preview" class="text-muted"/>
```

Comment:
Not sure why it's text-muted, makes it look strange in comparison to the rest of the form.